socorro/unittest/processor/test_mozilla_transform_rules.py

```diff
@@ -1351,7 +1329,18 @@ def test_bad_buildid(self):
         processor_meta = get_basic_processor_meta()
 
         rule = BetaVersionRule(config)
-        rule.act(raw_crash, raw_dumps, processed_crash, processor_meta)
+        with requests_mock.Mocker() as req_mock:
+            # NOTE(willkg): Is it possible to validate the buildid and
+            # reject it without doing an HTTP round-trip?
```

Comment:
I didn't bother dealing with this here since we don't often get malformed build ids in the crash report data. So it'll do an HTTP round trip--meh. The alternative is we write and maintain a bunch of code to validate build ids. Unenthusing.